src/sets.jl

```diff
@@ -30,6 +30,31 @@ julia> dimension(PositiveSemidefiniteConeTriangle(2))
 """
 function dimension end
 
+"""
+    dual_set(s::AbstractSet)
+
+Return the dual set of `s`, that is the dual cone of the set. This follows the 
+definition of `MathOptInterface` discussed in http://www.juliaopt.org/MathOptInterface.jl/stable/apimanual/#Advanced-1
```

Comment:
For version consistency, please reference this section using an internal link instead of a URL. See https://juliadocs.github.io/Documenter.jl/stable/man/syntax/#@ref-link-1. It might take a couple tries to work as I'm not sure of the exact syntax. You can double check that the link works by building the documentation locally.

Reply:
![image](https://user-images.githubusercontent.com/32756941/62068967-a55a7a80-b20d-11e9-8df7-fc95685a205d.png)